tests/microos/toolbox.pm

```diff
@@ -53,17 +64,19 @@ sub run {
 
 
     record_info 'Test', "Rootless toolbox as $user";
-    my $prefix = "runuser -l $user -c";
-    my $uid    = script_output "$prefix 'id -u'";
-    validate_script_output "$prefix 'toolbox -u id'", sub { m/uid=${uid}\(${user}\)/ }, timeout => 180;
-    die "$user shouldn't have access to /etc/passwd!" if (script_run("$prefix 'toolbox -u touch /etc/passwd'") == 0);
-    assert_script_run "$prefix \"podman rm toolbox-$user-user\"";
-
-    record_info 'Test',                                         "Rootfull toolbox as $user";
-    validate_script_output "$prefix 'toolbox -r id'",           sub { m/uid=0\(root\)/ };
-    assert_script_run "$prefix 'toolbox -r touch /etc/passwd'", fail_message => 'Root should have access to /etc/passwd!';
-    assert_script_run "podman rm toolbox-$user";
+    select_console 'user-console';
+    my $uid = script_output 'id -u';
+    validate_script_output 'toolbox -u id', sub { m/uid=${uid}\(${user}\)/ }, timeout => 180;
+    die "$user shouldn't have access to /etc/passwd!" if (script_run('toolbox -u touch /etc/passwd') == 0);
+
+    record_info 'Test',                               "Rootfull toolbox as $user";
+    validate_script_output 'toolbox -r id',           sub { m/uid=0\(root\)/ };
+    assert_script_run 'toolbox -r touch /etc/passwd', fail_message => 'Root should have access to /etc/passwd!';
+    assert_script_run 'podman ps -a';
+    clean_container_host(runtime => 'podman');
 
+    # Back to root
+    select_console 'root-console';
```

Comment:
i am not sure if you actually need this. Many modules start with select_console or select_serial_terminal. So do you really need it?

Reply:
yes, otherwise it will run the next commands in `$user` console...